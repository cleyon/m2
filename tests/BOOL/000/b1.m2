@comment	Simple boolean expressions
@define TRUE 1
@if defined(__TIME__) && TRUE
True
@else
False
@fi
@if defined(__TIME__) && FALSE
True
@else
False
@fi
