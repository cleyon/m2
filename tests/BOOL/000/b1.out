@booltest defined(__TIME__) && TRUE
@booltest defined(__TIME__) && FALSE
